platzky/engine.py

```
@@ -32,6 +44,9 @@
     def add_notifier(self, notifier):
         self.notifiers.append(notifier)
 
+    def add_cms_module(self, module: CmsModule):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)